crates/vm/levm/src/errors.rs

```diff
@@ -71,7 +71,7 @@ pub enum VMError {
     #[error("Transaction validation error: {0}")]
     TxValidation(#[from] TxValidationError),
     #[error("Offset out of bounds")]
-    OutOfOffset,
+    OutOfBounds,
```

Comment:
Totally agree! "Out of offset" sounds a bit strage.

I was actually messing around with this same error in my branch!